tools/mtouch/mtouch.csproj

```diff
@@ -1,5 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" ToolsVersion="4.0">
+  <Import Project="..\..\packages\XliffTasks.1.0.0-beta.20059.1\build\XliffTasks.props" Condition="Exists('..\..\packages\XliffTasks.1.0.0-beta.20059.1\build\XliffTasks.props')" />
+  <Import Project="..\..\packages\XliffTasks.1.0.0-beta.19606.1\build\XliffTasks.props" Condition="Exists('..\..\packages\XliffTasks.1.0.0-beta.19606.1\build\XliffTasks.props')" />
```

Comment:
Why two import statements? Isn't one enough?

Reply:
Not sure, I added the nuget using VSMac. I'll fix it.

Reply:
there are still two of them
please don't make comments as **resolved** since it means we can't double check if any action was done 
iow let the original commenter resolve his comments, based on your changes